mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -138,7 +143,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         showProgressDialog(Constants.PLEASE_WAIT);
         mEditProfilePresenter.fetchUserDetails();
-
+        Bitmap bitmap = load();
```

Comment:
* File being read on main thread.
* Business logic should not live in the activity but go into presenter.

Reply:
Done !